_posts/2021-06-10-solidity-0.8.5-release-announcement.md

```diff
@@ -0,0 +1,226 @@
+---
+layout: post
+published: true
+title: 'Solidity 0.8.5 Release Announcement'
+date: '2021-06-10'
+author: Solidity Team
+category: Releases
+---
+
+[Solidity v0.8.5](https://github.com/ethereum/solidity/releases/tag/v0.8.5) allows conversions
+from ``bytes`` to ``bytesNN`` values, adds the ``verbatim`` builtin function to inject
+arbitrary bytecode in Yul and fixes several smaller bugs.
+
+
+## Notable New Features
+### Bytes Conversion
+
+This release introduces the ability to convert ``bytes`` and ``bytes`` slices to fixed bytes types ``bytes1`` / ... / ``bytes32``.
+In case a byte array is longer than the target fixed bytes type, it will be truncated at the end:
+
+```solidity
+function f(bytes calldata c) public view returns (bytes8) {
+	return bytes8(c); // If c is longer than 8 bytes, truncation happens
+}
+```
+
+If the array is shorter than the target fixed type, it will be padded with zeros at the end:
+
+```solidity
+function f(bytes calldata c) public view returns (bytes8) {
+	require(c.length == 30, "");
+	bytes32 b = bytes32(c); // c is 30 bytes long, so b will be padded with two zero bytes at the end
```

Comment:
I think it would be better to have comments on next line. Helps us avoid the vertical scroll bar in source code. See: https://deploy-preview-141--solidity-blog.netlify.app/2021/06/10/solidity-0.8.5-release-announcement/